sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/WindowCallbackAdapter.java

```diff
@@ -16,11 +16,11 @@
 import org.jetbrains.annotations.Nullable;
 
 @Open
-class WindowCallbackAdapter implements Window.Callback {
+public class WindowCallbackAdapter implements Window.Callback {
```

Comment:
If you decide to drop `NextDrawListener` (see above), this can be package-private again